sleap/info/write_tracking_h5.py

```diff
@@ -286,12 +287,77 @@ def write_occupancy_file(
     print(f"Saved as {output_path}")
 
 
+def write_csv_file(output_path, data_dict):
+
+    """Write CSV file with data from given dictionary.
+
+    Args:
+        output_path: Path of HDF5 file.
+        data_dict: Dictionary with data to save. Keys are dataset names,
+            values are the data.
+
+    Returns:
+        None
+    """
+
+    if data_dict["tracks"].shape[-1] == 0:
+        print(f"No tracks to export in {data_dict['video_path']}. Skipping the export")
+        return
+
+    data_dict["node_names"] = [s.decode() for s in data_dict["node_names"]]
+    data_dict["track_names"] = [s.decode() for s in data_dict["track_names"]]
+    data_dict["track_occupancy"] = np.transpose(data_dict["track_occupancy"]).astype(
+        bool
+    )
+
+    # find frames with at least one animal tracked
```

Comment:
Nit: google style comments
```suggestion
    # Find frames with at least one animal tracked.
```